vg/vgtex/canvas_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 The gonum Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vgtex
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/plotter"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+)
+
+func Example() {
+	scatter, err := plotter.NewScatter(plotter.XYs{{1, 1}, {0, 1}, {0, 0}})
+	if err != nil {
+		log.Fatal(err)
+	}
+	p, err := plot.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	p.Add(scatter)
+	// p.HideAxes()
+	p.Title.Text = `A scatter plot: $\sqrt{\frac{e^{3i\pi}}{2\cos 3\pi}}$`
+	p.X.Label.Text = `$x = \eta$`
+	p.Y.Label.Text = `$y$ is some $\Phi$`
+
+	c := New(5*vg.Centimeter, 5*vg.Centimeter)
+	p.Draw(draw.New(c))
+	c.FillString(p.Title.Font, 2.5*vg.Centimeter, 2.5*vg.Centimeter, "x")
+
+	f, err := os.Create("testdata/scatter.tex")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	fmt.Fprintf(f, `%%%%%%
```

Comment:
SGTM.

I would propose:

```go
// New returns a new LaTeX canvas.
func New(w, h vg.Length) *Canvas

type Canvas struct {
  Standalone bool
  Header string
  // ...
}
```

and `vgtex.Canvas.WriteTo` would prepend `vgtex.Canvas.Header` only if `Standalone` is true. (and append `\end{document}\n`)